pages/architecture/certificates.rst

```diff
@@ -12,12 +12,41 @@ This does not give the foundation super powers since vendors and users (both val
 .. raw:: html
     :file: ../../_static/images/certificates.svg
 
-Extensions
-**********
+Vendor CA Certificate
+*********************
+
+Common Name: CN=...,O=...,C=NL
+
+=================================  ==========  =========================================
+Extension                          Critical?   Value
+=================================  ==========  =========================================
+BasicConstraints                   Yes         CA=true pathLenConstraint=1
+NameConstraints                    ???         ???
+KeyUsage                           Yes         digitalSignature & keyCertSign & crlSign
+SubjectAltName                     No          Vendor identifier: otherName, 1.3.6.1.4.1.54851.4=IA5String.<number>
+Nuts domain (1.3.6.1.4.1.54851.3)  No          IA5String=healthcare | social | pgo | insurance
+CRLDistributionPoints              No          Distribution point (URL) of the CRL
+=================================  ==========  =========================================
+
+TLS Client Certificate
```

Comment:
Between Vendor CA and this one, the TLS CA is missing

Reply:
True, I'll add it shortly